AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/Statement+Model.swift

```diff
@@ -105,6 +105,8 @@ extension Statement: StatementModelConvertible {
                 }
             }
         }
+        modelDictionary["__typename"] = modelSchema.name
```

Comment:
why do we need to do this?

Reply:
Flutter engineer noticed that we were not populating this field when retrieving models from the local DB